components/tabs/tabs.js

```diff
@@ -393,37 +397,77 @@ class Tabs extends LocalizeCoreElement(ArrowKeysMixin(SkeletonMixin(LitElement))
 		return this.shadowRoot.querySelector('.d2l-tabs-container-list').getBoundingClientRect();
 	}
 
+	hideTab(tab) {
+		tab.setAttribute('data-state', 'removing');
+		return (Object.keys(this._tabIds).length > 1 && !reduceMotion) ? this._animateTabRemoval(tab) : Promise.resolve();
+	}
+
 	#checkTabPanelMatchRequested;
 	#panels;
 	#updateAriaControlsRequested;
 
-	_animateTabAddition(tabInfo) {
+	_animateTabAddition(tab) {
+		if (!tab || reduceMotion) return Promise.resolve();
```

Comment:
There's a slight difference in the two paths here, in that we call `tab.setAttribute('data-state', '');` and `this.requestUpdate()` in one case but not the other. So this code is assuming that the consumer will not have set `adding` if the user prefers reduced motion. Maybe that's fine if everything is being controlled by the caller - currently, this `Promise.resolve` will never be called. Sorry if my previous comment has added confusion.

Reply:
Btw, I think it is good that it doesn't go into the "adding" state when the user prefers reduced motion. I'd worry that that would cause some unwanted styles to get applied or something. In that case, it should just skip that "adding" state the way you have it.